src/views/VideoMania/index.jsx

```
@@ -133,27 +137,45 @@ const VideoMania = () => {
               <div className="heading">{video?.sub_category_name}</div>
               <div className="video-card-container">
                 {video?.video_result?.Videos?.map((video) => (
-                  <VideoCard video={video} />
+                  <VideoCard
+                    video={video}
+                    onClick={() => {
+                      setCurrentVideo(video);
+                      setVideoModal(true);
+                    }}
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)